critiquebrainz/frontend/forms/profile.py

```diff
@@ -13,3 +13,8 @@ class ProfileEditForm(Form):
         validators.Optional(strip_whitespace=False),
         validators.Email(message=lazy_gettext("Email field is not a valid email address."))])
     show_gravatar = BooleanField(lazy_gettext("Show my Gravatar"))
+    license_choice = RadioField(lazy_gettext("Preferred License Choice"),
+        choices=[
+            ('CC BY-SA 3.0', lazy_gettext('Allow commercial use of this review(<a href="https://creativecommons.org/licenses/by-sa/3.0/" target="_blank">CC BY-SA 3.0 license</a>)')),  # noqa: E501
```

Comment:
Really nitpicking now :sweat_smile:. But there should be space between review and the opening bracket. :).